src/akamai/js/akamaiHD_flash.js

```diff
@@ -242,6 +243,16 @@ require("../../../html5-common/js/utils/constants.js");
      * @param {boolean} rewind True if the stream should be set to time 0
      */
     this.load = function(rewind) {
+      if (loaded && !rewind) return;
+      if (!!rewind) {
+        try {
+          this.callToFlash("load("+rewind+")");
+          loaded = true;
+        } catch (ex) {
+          // error because currentTime does not exist because stream hasn't been retrieved yet
+          OO.log("[Akamai HD]: Load URL", "Failed to rewind video, probably ok; continuing");
+        }
+      }
```

Comment:
@angieisrani: Yes. Also I have never seen rewind coming false from VTC. This is true even in case of initial set time. Same implementation you can see even in html5 video plugin